example/App.js

```diff
@@ -9,6 +9,9 @@
 
 import React, {Component} from 'react';
 import {Platform, StyleSheet, Text, View} from 'react-native';
+import NetInfo from '@react-native-community/netinfo';
+
+console.log("NetInfo", NetInfo)
```

Comment:
Oops, can you remove that one?

Reply:
This is removed is a subsequent commit. Would it best for me to squash the commits together?

Reply:
Oh nevermind, you don't have to, I just didn't realize this as I looked at all the commits in isolation.

Reply:
Yes, I thought that was the case. I'll squash down anyway. Keeps things nice and tidy.